src/components/catalog/CatalogPage.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import CatalogList from './CatalogList';
+import CatalogFilter from './CatalogFilter';
```

Comment:
Use the `index.js` to combine your imports.
```suggestion
import { CatalogFilter, CatalogList } from '..';
```